apps/backend/src/datasources/FapDataSource.ts

```diff
@@ -71,11 +71,14 @@ export interface FapDataSource {
     userId: number,
     proposalPk: number
   ): Promise<boolean>;
-  getFapProposals(filter: {
-    fapId: number;
-    callId?: number | null;
-    instrumentId?: number | null;
-  }): Promise<FapProposal[]>;
+  getFapProposals(
+    filter: {
+      fapId: number;
+      callId?: number | null;
+      instrumentId?: number | null;
+    },
+    currentUerId?: number
```

Comment:
```suggestion
    currentUserId?: number
```

Spell check. But I am still trying to figure out the best way to rename for making it more self documented